project/Dependencies.scala

```diff
@@ -11,11 +11,12 @@ object Dependencies {
   object Versions {
     // Version numbers
 
-    val Scala210 = "2.10.7"
     val Scala212 = "2.12.10"
     val Scala213 = "2.13.1"
     val Scala    = Seq(Scala212, Scala213)
-    val SbtScala = Seq(Scala212, Scala210)
+    val SbtScala = Seq(Scala212)
+
+    val Sbt1 = "1.2.8"
```

Comment:
Should we call this `TargetSbt1`, or something similar, perhaps?

Reply:
Done. Left a comment there as well.